scripts/scripts.js

```diff
@@ -716,18 +716,52 @@ export async function fetchBlogArticleIndex() {
   return (index);
 }
 
+/**
+ * forward looking *.metadata.json experiment
+ * fetches metadata.json of page
+ * @param {path} path to *.metadata.json
+ * @returns {Object} containing sanitized meta data
+ */
+
+async function getMetadataJson(path) {
+  const resp = await fetch(path.split('.')[0]);
+  const text = await resp.text();
+  const headStr = text.split('<head>')[1].split('</head>')[0];
+  const head = document.createElement('head');
+  head.innerHTML = headStr;
+  const metaTags = head.querySelectorAll(':scope > meta');
+  const meta = {};
+  metaTags.forEach((metaTag) => {
+    const name = metaTag.getAttribute('name') || metaTag.getAttribute('property');
+    const value = metaTag.getAttribute('content');
+    if (meta[name]) {
+      meta[name] += `, ${value}`;
+    } else {
```

Comment:
it seems like a long shot, but it is possible that the content of a meta tag would have a comma in it? description comes to mind, but there is only one meta description... but is it possible that a tag that appears multiple times (like `article:tag`) to contain a comma? would that mess up the ability to parse this field when it is returned?

Reply:
i think that's possible, having said that though, from an authoring standpoint i think the `tags` are entered comma separated to begin with and i would expect a server-sided function to just return those verbatim. 
so i should actually do some more reversing, and get back to the name value pair that was authored as part of the `metadata` table to simulate what i would expect from the `metadata.json` including for example deduping `og:image`, `og:image:secure`, `twitter:image` into `image`, but i just wanted to get it working.

more generally i think the consuming application knows what types of values it expects for which metadata field, so in this case it would only `split(',')` on `article:tag` but not on something like the `description` which is likely to contain commas as well.